### PR TITLE
Fix backward compatibility problems with last wp-source release

### PR DESCRIPTION
**Pull request by luisherranz**
#### Description of what you did:

- I have added a fallback to 0 to make sure they are always a number.
- I have fixed a wrong set of the total pages when there are items and the headers are missing.
- I have fixed a backward compatibility problem with our internal handlers.

#### My PR is a:

- 🐞 Bug fix

#### Is the PR ready to be merged?

- ✅ Yes!

### CODE REVIEW

**Comment by luisherranz** [comment]
Ready for review but merge blocked by https://github.com/frontity/frontity/pull/338.

**Comment by luisherranz** [comment]
Blocked until I review the logic when taxonomies exist but don't have any post.

**Comment by luisherranz** [comment]
After reviewing the logic with @DAreRodz it looks like the logic was matching WordPress behavior, which is a bit weird because it returns 0 pages for taxonomies without items, instead of 1 page. So I guess it's fine to match what WordPress does... 🤷‍♂️

Ready for review!

**Review by michalczaplinski — APPROVED** [review]
LGTM 👍

File: .changeset/early-goats-appear.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+"@frontity/wp-source": patch
+---
+
+- Fix wrong default totals when headers are missing. It was set to 0, but it has to be 1.
```
Comment (michalczaplinski) [inline .changeset/early-goats-appear.md]: Is this correct? Because in the code, I can see that the fallback is still `0`.

**Review by michalczaplinski — APPROVED** [review]
I just have the one comment. Not sure if it's legitimate or not. If not then approved 👍

File: .changeset/early-goats-appear.md (hunk 1 above)
Comment (luisherranz) [reply .changeset/early-goats-appear.md]: Good catch, thanks!

**Pull request merged**